src/transforms.jl

```diff
@@ -170,16 +170,17 @@ reapply(transform::StatelessFeatureTransform, table, cache) =
 # ------------------
 
 function applyfeat(transform::ColwiseFeatureTransform, feat, prep)
-  # basic checks
-  for assertion in assertions(transform)
-    assertion(feat)
-  end
-
   # retrieve column names and values
   cols   = Tables.columns(feat)
   names  = Tables.columnnames(cols)
   snames = choose(transform.colspec, names)
-
+  selected_cols = feat |> Select(snames)
```

Comment:
this is not following the code style of the surrounding code. can you please avoid underscores and choose a short and yet descriptive name to represent the selection? also this is missing a new line before the next comment

Reply:
Okay, maybe I can keep that as scols

Reply:
the result is a table of selected features so maybe sfeat would be more appropriate

Reply:
Okay, now it is done.